tasks/basecalling/task_dorado_demux.wdl

```diff
@@ -0,0 +1,95 @@
+version 1.0
+
+task dorado_demux {
+  input {
+    Array[File] bam_files
+    String kit_name
+    String fastq_file_name
+    Int cpu = 4 
+    Int memory = 16
+    Int disk_size = 50
+    String docker = "us-docker.pkg.dev/general-theiagen/staphb/dorado:0.8.0"
+  }
+
+  command <<< 
+    set -e
```

Comment:
might be good to adjust this to `set -euo pipefail` as there's one pipe used below